Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -1449,3 +1461,22 @@ theorem hasFPowerSeriesAt_iff' :
 #align has_fpower_series_at_iff' hasFPowerSeriesAt_iff'
 
 end
+
+/-!
+### Combining analytic functions: sums, products, and pairs
+-/
+
+/-- Finite sums of analytic functions are analytic -/
+theorem AnalyticAt.sum {f : ℕ → E → F} {c : E} (h : ∀ n, AnalyticAt 𝕜 (f n) c) (N : Finset ℕ) :
```

Comment:
Does this generalize to a Finset of an arbitrary type?

Reply:
Done.